gke/mcs.yaml

```diff
@@ -17,6 +17,8 @@ kind: MultiClusterService
 metadata:
   name: website-mcs
   namespace: website
+  annotations:
+    cloud.google.com/backend-config: '{"ports": {"8080":"backendconfig"}}'
```

Comment:
This is supposed to refer to the yaml? worth commenting so?

Reply:
Done